packages/e2e/lib/_extension_io.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// The dart:io implementation of [registerWebServiceExtension].
+///
+/// See also:
+///
+///  * [_extension_web.dart], which has the dart:html implementation
+void registerWebServiceExtension(
+    Future<Map<String, dynamic>> Function(Map<String, String>) call) {
+  throw UnsupportedError('Use registerServiceExtension instead');
```

Comment:
Is this error message correct? We either call both `registerWebServiceExtension` _and_ `registerServiceExtension`, or we only call `registerServiceExtension`.

If I'm reading the code correct the error should read something like "registerWebServiceExtension is only available on the Web".

Reply:
done.